src/py/main.py

```diff
@@ -52,6 +52,8 @@ def main():
     exit(-1)
 
   eng_name = raw_input("What is the name of the person we are onboarding? ")
+  start_date = raw_input("What date does the person start? (e.g. 2017-01-24) ")
+  eng_name += " (Start " + start_date + ") "
```

Comment:
this works for this particular use-case, but i'm worried about just shoving the date into the name field because this project advertizes `{{name}}` as a variable that you can use in configs in places beyond just the Epic name.  I'd prefer if we could capture this date string and pass it into the rest as another `{{start_date}}` variable that can be used, and it can be concatenated as desired in the config file itself.